src/clipboard.js

```diff
@@ -86,10 +86,10 @@ class Clipboard extends Emitter {
      */
     static isSupported(action = ['copy', 'cut']) {
         const actions = (typeof action === 'string') ? [action] : action;
-        let support = !!document.queryCommandSupported;
+        let support = !document.queryCommandSupported;
```

Comment:
There likely should be

```js
'queryCommandSupported' in document && typeof document.queryCommandSupported === 'function'
```

Reply:
Why make it more complicated again? I would keep `!!` to convert it to a boolean.

Reply:
It's not that complicated, but make check more obvious

Reply:
I don't think that this should be part of this refactor / change PR but another.

Reply:
You right